src/WebJobs.Extensions.DurableTask/LinuxLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    public class LinuxLogger : ILogger
+    {
+        private readonly bool inConsumption;
+        private readonly string consolePrefix = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+        private readonly string loggingPath = "/var/log/functionsLogs/durableevents.log";
+
+        public LinuxLogger(bool inConsumption)
+        {
+            this.inConsumption = inConsumption;
+        }
+
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            // TODO: I this we can leave this unimplemented
+            throw new NotImplementedException();
+        }
+
+        public bool IsEnabled(LogLevel logLevel)
+        {
+            // TODO: assuming these logs should never be disabled
+            return true;
```

Comment:
I would say we only want it enabled if we're on an App Service Linux platform. But then again, the other way you could do it (and maybe you are, I haven't gone through the whole PR yet) is to just not add this logger at all if we're not on App Service Linux.